theano/sandbox/cuda/dnn.py

```diff
@@ -2597,7 +2597,8 @@ def local_softmax_dnn_grad(node):
                 gpu_contiguous(ins[0]),
                 gpu_contiguous(ins[1])
             )
-            return [out.dimshuffle(0, 1)]
+            return [out.dimshuffle(0, 2)]
+            
```

Comment:
remove this blanc line. This make a test fail at this isn't pep8 compliant. At the same time, can you remove the merge commit?